CONTRIBUTING.md

```diff
@@ -119,3 +121,13 @@ Examples:
 
   BREAKING CHANGE: `extends` key in config file is now used for extending other config files
   ```
+
+#### Releases
+
+Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to `v1.3.0` and outputs the results (`--verbose`):
```

Comment:
```suggestion
Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md). We recomment using [`bump2version`](https://github.com/c4urself/bump2version) instead of `tbump`, however. The example below updates the release to version `1.3.0` and outputs the results (`--verbose`):
```